src/common/tracer.h

```diff
@@ -0,0 +1,66 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
```

Comment:
where are these defines used?

Reply:
I was facing some jaeger related issue and when searched for solution this thing worked

Reply:
you still need a header guard, you can use `#pragma once`

Reply:
yeah sorry TRACER_H_ serves as header guard, got confused.
`htons` are required by `jaeger` libraries I was facing issue compiling with jaeger so this solution I got from stack overflow.